### ISSUE TITLE
Sync up the collection information with Chromebook app

### ISSUE THREAD

**Issue by vanessa-chang** [endlessm/kolibri-app#146]
In the latest beta apps, the windows collection is different from the chromebook's, such as the featured channels and the new addded channel are not seen the Windows's one.

The collection needs to be updated and sync with the one we used in Chromebook. 

> StarNight
Notice the build scripts:
Windows app: Use the latest release version of endless-key-collections https://github.com/endlessm/endless-key-app/blob/v6.31.2/.github/workflows/main.yml#L103-L109
Chromebook app: Use main branch's latest commit of endless-key-collections https://github.com/endlessm/kolibri-installer-android/blob/4bab8c8c92c0ba1b6ce9b748a2845b92407f9153/Makefile#L142-L145
erikos
So there are some unreleased changes in the collections, if the Windows app pulls in the released collections information that would explain the difference.
erikos
See the latest changes from Heather where she added a fourth featured channel; https://github.com/endlessm/endless-key-collections/commit/6b88342e513e0e24e0772c7c3cebcf1801a8f975
erikos
So that is expected.
StarNight
catching latest modification is the expected one!

**Comment by wjt** [comment]
I stumbled upon this ticket again today:

- https://github.com/endlessm/kolibri-explore-plugin/issues/610

So rather than building the collections into the app, we would fetch them from the web at runtime. I have advocated for this in the past. The problem with that is that in the Endless OS version of the app, we plan to preinstall the content but also run the welcome flow and allow the user to choose their content pack. This wouldn't cause anything to be downloaded, but would adjust what is shown on the Discovery tab.

----

The other interesting quirk here is: if we change the Chrome OS app to only use the released version of endless-key-collections, it becomes much harder to test changes to the collections without releasing them…
